Kuroba/app/build.gradle

```diff
@@ -290,6 +290,7 @@ dependencies {
     implementation "androidx.appcompat:appcompat:${versions.appcompat}"
     implementation 'androidx.browser:browser:1.2.0'
     implementation 'androidx.constraintlayout:constraintlayout:2.0.1'
+    implementation 'androidx.preference:preference:1.1.1'
```

Comment:
Why did you add this dependency, btw?

Reply:
It was needed to fix the Preference deprecation in the second commit.

Reply:
Ohh I see.